### PR TITLE
metasploit cheat-sheet

### PR DESCRIPTION
**Pull request by pseudozidu**
**What does your Instant Answer do?**
cheatsheet goodies

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.cheatography.com/huntereight/cheat-sheets/metasploit-4-5-0-dev-15713/

**Why did you choose this data source?**
It is cool cheat sheet and most people used it

**Are there any other alternative (better) data sources?**
yes there many data source, but it is good and descriptive compare to other.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
security, sysadmin

**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**
Yes, it is cheatsheet for security professional and pen tester.

**Are you having any problems? Do you need our help with anything?**
No

**Where did you hear about DuckDuckHack? (For first time contributors)**
i am co-organizer for Delhi group quack and hack & it will example for them.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
I have added terminal template and working attached old screenshots.

image

-----
IA Page: https://duck.co/ia/view/metasploit_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/metasploit.json [hunk 1]
```diff
@@ -0,0 +1,444 @@
+{
+	    "id": "metasploit_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/metasploit.json]: nit: the indentation is off here. Please make sure it's indented properly with 4 spaces per indent.

File: share/goodie/cheat_sheets/json/metasploit.json [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+	    "name": "Metasploit",
+	    "description": "A framework that provides information about security vulnerabilities",
+	    "metadata": {
+	        "sourceName": "Cheatography",
+	        "sourceUrl": "http://www.cheatography.com/huntereight/cheat-sheets/metasploit-4-5-0-dev-15713/"
+	    },
+	    "template_type": "terminal",
+	    "section_order": [
+	        "database commands",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/metasploit.json]: Let's use proper Title Casing for the section headers:

```
Database Comands
Core Commands
Meterpreter Core and File System
...
```

File: share/goodie/cheat_sheets/json/metasploit.json [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+	        {
+	            "val": "show metasploit info",
+	            "key": "version"
+	        }],
+	        "Meterpreter Core and File System": [{
+	            "val": "background the current session",
+	            "key": "background"
+	        },
+	        {
+	            "val": "kill a background meterp¬reter script",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/metasploit.json]: Typo: there's a `¬` in the middle of this

File: share/goodie/cheat_sheets/json/metasploit.json [hunk 4]
```diff
@@ -0,0 +1,444 @@
...
+	        {
+	            "val": "drop into a system command shell",
+	            "key": "shell"
+	        },
+	        {
+	            "val": "shuts down the remote computer",
+	            "key": "shutdown"
+	        },
+	        {
+	            "val": "attempt to steal an impers¬onation token from the process",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/metasploit.json]: Same here with the `¬`

File: share/goodie/cheat_sheets/json/metasploit.json [hunk 5]
```diff
@@ -0,0 +1,444 @@
...
+	        "Meterpreter Priv Commands": [{
+	            "val": "list webcams",
+	            "key": "webcam_list"
+	        },
+	        {
+	            "val": "take a snapshot from the specified webcam",
+	            "key": "webcam_snap"
+	        },
+	        {
+	            "val": "attempt to elevate your priviledge to that of local system",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/metasploit.json]: typo: privilege

**Comment by mintsoft** [comment]
@pseudozidu The first character on every line is still a hard-tab, the `expand` command in linux will convert it to spaces for you?

File: share/goodie/cheat_sheets/json/metasploit.json [hunk 6]
```diff
@@ -0,0 +1,456 @@
...
+    "template_type":"terminal",
+    "section_order":[  
+        "Database Commands",
+        "Core Commands",
+        "Meterpreter Core and File System",
+        "Meterpreter User Interface Commands",
+        "Meterpreter System Commands",
+        "Meterpreter Priv Commands"
+    ],
+    "Sections":{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/metasploit.json]: Careful! No capital letters :)

**Comment by moollaza** [comment]
LGTM :+1: 

I'll fix the capital S, it's currently breaking the IA :+1:

**Comment by pseudozidu** [comment]
Thanks @moollaza

**Comment by moollaza** [comment]
@pseudozidu congrats! It's live: https://duckduckgo.com/?q=slack+cheat+sheet&t=ffab&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**